ldi-core/sparql-construct/pom.xml

```diff
@@ -8,4 +8,14 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>sparql-construct</artifactId>
+    <properties>
+        <jena-geosparql.version>4.7.0</jena-geosparql.version>
```

Comment:
do we need these properties? Since we are using ${jena.version} everywhere instead